python/BioSimSpace/Sandpit/Exscientia/Trajectory/_trajectory.py

```diff
@@ -202,9 +208,24 @@ def getTrajectory(self, format="mdtraj"):
                The trajectory in MDTraj or MDAnalysis format.
         """
 
-        if format.upper() not in ["MDTRAJ", "MDANALYSIS"]:
+        # Strip whitespace and convert to upper case.
+        format = format.replace(" ", "").upper()
+
+        if format.replace(" ", "").upper() not in ["MDTRAJ", "MDANALYSIS", "AUTO"]:
             _warnings.warn("Invalid trajectory format. Using default (mdtraj).")
-            format = "mdtraj"
+            format = "MDTRAJ"
```

Comment:
Would it be best to default to `AUTO`

Reply:
Yes, it could be, I was just being consistent with the existing behaviour so as not to surprise anyone. I can update and check that it doesn't break anything.

Reply:
@lohedges So currently, if my understanding is correct, when backend is set to auto, it will try to use MDTRAJ first. So unless the mdtraj failed, I would imagine that it would produce the same result?